Package.swift

```diff
@@ -32,6 +32,10 @@ let package = Package(
             url: "https://github.com/Basis-Theory/basistheory-ios",
             from: "2.5.0"
         ),
+        .package(
+            name: "Sentry", 
+            url: "https://github.com/getsentry/sentry-cocoa", 
+            from: "6.19.7"),
```

Comment:
This is the version we use in payments-api and snap payments. Let me know if we would need to use another version

Reply:
can we use the version recommended by Sentry ios docs https://docs.sentry.io/platforms/apple/guides/ios/#install:~:text=tag%20%3D%3E%20%27-,8.9.1,-%27%0Aend

Reply:
"9.0" as suggested in the docs did not seem to be supported. I used 8.8.0 which I did see used in other sdk repos 7e058b4102eb99776714bfcd69b9723c9836f032